runtime/runtime/src/tests/mod.rs

```diff
@@ -38,6 +44,22 @@ fn create_receipt_with_actions(
     })
 }
 
+/// Sets the given gas and compute costs for sha256 in the runtime config.
+/// Sets the rest of the costs to free. Returns the sha256 cost.
+fn set_sha256_cost(
```

Comment:
extracted this common code since it allows me to test cases where one function passes gas limit and another function exceeds.